osi_lowleveldata.proto

```diff
@@ -1,147 +0,0 @@
-syntax = "proto2";
```

Comment:
Removing the LowLevelData messages might affect the ability to faithfully interpret stored data, which we guarantee for minor releases (quoting from our README):

>Backward compatibility:
>Definition: "A newer version of code can be used to read old files"
>All files that have been recorded in the past with a specific major version are FAITHFULLY valid with all combinations of
>higher minor and patch versions of the same major version.

where faithfully is defined as
>FAITHFULLY "All recorded data is correctly interpreted by the interface"

Personally I think that for the low-level-data there was not much usage until now, so breaking this promise seems valid, however given also the lane changes we should think about just releasing a 3.0.0 version? @CarloVanDriestenBMW what do you think?